lib/Controller/SignalingController.php

```diff
@@ -136,29 +136,32 @@ public function getSettings(string $apiVersion, string $token = ''): DataRespons
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
-		$stun = [];
-		$stunServer = $this->talkConfig->getStunServer();
-		if ($stunServer) {
-			$stun[] = [
-				'url' => 'stun:' . $stunServer,
-			];
+		$stunUrls = [];
+		$stunServers = $this->talkConfig->getStunServers();
+		foreach ($stunServers as $stunServer) {
+			$stunUrls[] = 'stun:' . $stunServer;
 		}
+		$stun = [
+			'urls' => $stunUrls
```

Comment:
Can you bump the api version for signaling and create different arrays based on the already available `$apiVersion`?